.vscode/cspell.json

```diff
@@ -36,6 +38,7 @@
     // Spelling issues must be addressed before adding these back in.
     "sdk/agrifood/*/api/*.cs",
     "sdk/communication/*/api/*.cs",
+    "sdk/confidentialledger/Azure.Security.CodeTransparency/tests/TestFiles/*",
```

Comment:
I think you'd want to move this up above the comment.   The test files aren't part of your public API and you likely won't be fixing them.

Reply:
moved above the comment